docs/source/docs/objectDetection/about-object-detection.md

```diff
@@ -34,7 +35,51 @@ The same area, aspect ratio, and target orientation/sort parameters from {ref}`r
 
 ## Training Custom Models
 
-Coming soon!
+You can build your own YOLOv5 model for use with PhotonVision's Object Detection. However, this process is complex and requires a good understanding of machine learning, but it can be done. Here's an overview of the process.
+
+### Step 1: Model Training
+
+You can train the Object Detection model using [Ultralytics' YOLOv5 repository](https://github.com/ultralytics/yolov5), then use [airockchip's YOLOv5 fork](https://github.com/airockchip/yolov5) to export the model currectly before the export to a deployable RKNN file. The specific training setup will vary depending on your dataset and configuration, but here’s an example using `yolov5s`. Other models should work as well. You **will** need [PyTorch](https://pytorch.org/get-started/locally/). This example has been tested on Ubuntu 22.04.
+
+#### Downloading nessessary files
+
+```bash
+git clone https://github.com/ultralytics/yolov5.git
+git clone https://github.com/airockchip/yolov5.git airockchip-yolov5
+wget https://gist.githubusercontent.com/Alex-idk/9a512ca7bd263892ff6991a856f1a458/raw/e8c0c9d8d5a1a60a2bbe72c065e04a261300baac/onnx2rknn.py # This is the onnx to rknn convertion script
```

Comment:
Script looks reasonable, seems like the same thing most online resources have been hinting at.

Do we wanna pull this into the photonvision repo proper?

Reply:
As part of this PR, commit this file to our scripts folder or something in the monorepo.

Reply:
It is inside the scripts folder, however I did update the script but haven't tested it yet.